js/script.js

```diff
@@ -29,13 +31,13 @@ function resizeHandler() {
 function scrollifySection() {
     if ($(window).width() > 320) {
         $.scrollify({
-            interstitialSection : ".footer, .hungry",
+            interstitialSection : ".footer, .hungry, .section:first",
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/:first ja https://api.jquery.com/first-selector/. :first ei siis ole W3C:n mukainen selektori, suosittelen käyttämään esim. first-of-type jos first-child ei toimi.